modules/apps/search/search-web/src/META-INF/resources/view.jsp

```diff
@@ -57,7 +57,13 @@ pageContext.setAttribute("portletURL", portletURL);
 		</aui:select>
 
 		<aui:field-wrapper inlineField="<%= true %>">
-			<aui:button cssClass="search-button" icon="icon-search" onClick='<%= renderResponse.getNamespace() + "search();" %>' value="search" />
+			<a class="btn btn-default" href="javascript:;" onclick="<%= renderResponse.getNamespace() + "search();" %>">
```

Comment:
This doesn't seem very semantic. Why don't we just use a regular button?
Because ultimately, all that search function is doing is trying to prevent an empty search from going through. But if we use an anchor, and the JS errors for any reason (or is just slow to load), the submit won't go through.